Dockerfile.ocp

```diff
@@ -4,8 +4,14 @@ COPY . .
 RUN make build
 RUN make tools
 
+COPY ./config/crd/ocp/ocp_kustomization.yaml /go/src/github.com/metal3-io/baremetal-operator/config/crd/kustomization.yaml
+RUN make manifests
+COPY /go/src/github.com/metal3-io/baremetal-operator/config/render/capm3.yaml /go/src/github.com/metal3-io/baremetal-operator/manifests/0000_30_baremetal-operator_01_baremetalhost.crd.yaml
```

Comment:
I think this is going to suffer from the same problem, COPY's first argument is the file system outside the container.

Reply:
Yes, fixed that and also pre-created /manifests.